reddit/data.go

```diff
@@ -35,6 +37,12 @@ type Comment struct {
 	Distinguished string `mapstructure:"distinguished"`
 }
 
+// IsRoot is true when the comment is a top level comment.
+func (c *Comment) IsRoot() bool {
```

Comment:
Definitively. I took the name from the python based wrapper and thought they maybe had their reasons for the naming but I can of course change it.

https://github.com/praw-dev/praw/blob/master/praw/models/reddit/comment.py#L16